blog/blogpost/blogpost_anatomy_3d.md

```diff
@@ -0,0 +1,105 @@
+
+## Conclusion
+Our goal was straightforward: create a game that blends entertainment with education. And we are proud to say we achieved it. We have crafted a platform that introduces young learners to the complexities of human anatomy in a way that sticks. By leveraging 3D technology and gamification, we have transformed intricate anatomical concepts into interactive challenges that foster learning through play.
+
+From designing the frontend interface to architecting the backend infrastructure, every aspect of this project demanded precision and a willingness to push boundaries. Of course, it wasn't all smooth sailing. We encountered our fair share of obstacles along the way – from sourcing suitable 3D models to debugging intricate code. But with each challenge came an opportunity for growth, pushing us to refine our skills and find innovative solutions.
+
+As we reflect on our accomplishments, we are excited about the impact **Anatomy 3D** will have on educational practices. By making learning accessible, interactive, and engaging, we aim to inspire a new generation of learners to explore the intricacies of the human body.
+
+### Possible future work and improvements
+
+Our application is a complete game, containing all the features we intended to include. Looking ahead, we envision possibilities for refining its performance and introducing innovative features beyond the existing framework. In particular, we are thinking about
+
+- Enhancing our learning game involves refining the 3D models to accurately represent real-world objects and to enhance their visual appeal. This improvement can be achieved by either securing funds to obtain higher-quality models or by partnering with skilled 3D model designers. By investing in improved models, we aim to provide users with a more immersive and engaging experience, helping them better understand the concepts being taught within the game. Additionally, enhancing the aesthetics of the models can make the learning process more enjoyable and memorable for our audience, ultimately contributing to the effectiveness of the educational experience.
+- Expanding our learning game involves introducing new game levels, possibly focusing on topics like the nervous system and muscles. By incorporating these additional levels, we aim to provide users with a deeper understanding of complex biological concepts in a fun and interactive way. These new levels could offer challenges and puzzles related to the functioning of the nervous system and the mechanics of muscles, allowing players to explore these topics in more detail while advancing through the game. Furthermore, by diversifying the content, we can cater to a wider range of learners and keep them engaged for longer periods, ultimately enhancing the educational value of the game.
+
+## Team members
+- Rebeca Morales Masis
+- Edilbert Christhuraj
+- Mentor: Jose Riegel
```

Comment:
typo of my name :'), haha

Reply:
Fixed it - Rigel is correct - right?

Reply:
ah right, sorry looked at the wrong commit. :+1: